lib/components/src/navigation/routed_link.test.js

```diff
@@ -6,89 +6,132 @@ const LEFT_BUTTON = 0;
 const MIDDLE_BUTTON = 1;
 const RIGHT_BUTTON = 2;
 
+const createEvent = options => ({
+  button: LEFT_BUTTON,
+  preventDefault: jest.fn(),
+  ...options,
+});
+const render = props => shallow(<RoutedLink {...{ children: 'Content', ...props }} />);
```

Comment:
I feel like this is unnecessarily obtuse @ndelangen. IMO we should follow idiomatic React to make it easy for contributors.